apps/preview/nuxt/pages/showcases/Filters/Brand.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <SfAccordionItem v-model="open" class="w-full md:max-w-[376px]">
+    <SfListItem
+      v-for="{ id, value, label, counter } in details"
+      :key="id"
+      tag="label"
+      size="sm"
+      :class="['px-1.5 bg-transparent hover:bg-transparent', { 'font-medium': isItemActive(value) }]"
+    >
+      <template #prefix>
+        <SfCheckbox v-model="selectedFilters" :disabled="counter === 0" :value="value" />
+      </template>
+      <p>
+        <span class="mr-2 text-sm">{{ label }}</span>
```

Comment:
Label for disabled one should be gray and they look like they aren't centred
<img width="223" alt="Screenshot 2023-05-31 at 08 36 12" src="https://github.com/vuestorefront/storefront-ui/assets/46591755/2f5ddbbd-4959-41dd-b5ff-03524be47e8a">

Reply:
thx, fixed